lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -141,8 +170,9 @@ public ContextRuntime SpawnChildContext(IConfiguration contextConfiguration, ICo
                 }
                 try
                 {
-                    IInjector childServiceInjector = _serviceInjector.ForkInjector(serviceConfiguration);
-                    var childContext = new ContextRuntime(childServiceInjector, contextConfiguration, Optional<ContextRuntime>.Of(this));
+                    var childServiceInjector = _serviceInjector.ForkInjector(serviceConfiguration);
```

Comment:
Then you might want to name serviceConfiguration as childServiceConfiguration to make it clear.